README.md

```diff
@@ -484,6 +484,7 @@ Table of Contents
   * [Yandex.Connect](https://connect.yandex.com/pdd/) — Free email and DNS hosting for up to 1,000 users
   * [yopmail.fr](http://www.yopmail.fr/en/) — Disposable email addresses
   * [zoho.com](https://www.zoho.com/mail/) — Free Webmail and collaboration for up to 5 users
+  * [emailprofi.cz](https://emailprofi.cz) — Free email hosting
```

Comment:
cant seem to find a english version of the UI? I can unfortunately not vet what I cant read :(

Reply:
> cant seem to find a english version of the UI? I can unfortunately not vet what I cant read :(

Unfortunately the interface is only in Czech.

Reply:
OK, appreciate the contribution but would be tough to maintain it on the list so unfortunately will have to pass on this one